package.json

```diff
@@ -78,17 +78,19 @@
     "detect-node": "^2.0.3",
     "hapi": "^16.6.2",
     "hat": "0.0.3",
+    "ipfs-api": "^17.3.0",
```

Comment:
While you are at it, mind upgrading ipfs-api to latest?
<img width="253" alt="image" src="https://user-images.githubusercontent.com/1211152/36475815-d3828044-16f3-11e8-9bbc-8197a49a218e.png">